src/App.tsx

```diff
@@ -1,51 +1,85 @@
 import React from 'react';
 import './App.scss';
 
-// import goodsFromServer from './goods';
-
-export const App: React.FC = () => {
-  return (
-    <main className="App">
-      <header className="App__header">
-        <h1 className="App__title">
-          Jam is selected
-        </h1>
-
-        <button
-          type="button"
-          className="App__clear"
-        >
-          Clear
-        </button>
-      </header>
-
-      <ul>
-        <li className="Good">Dumplings</li>
-        <li className="Good">Carrot</li>
-        <li className="Good">Eggs</li>
-        <li className="Good">Ice cream</li>
-        <li className="Good">Apple</li>
-        <li className="Good">Bread</li>
-        <li className="Good">Fish</li>
-        <li className="Good">Honey</li>
-        <li className="Good Good--active">Jam</li>
-        <li className="Good">Garlic</li>
-      </ul>
-
-      {/* Put required buttons into each Good */}
-      <button
-        type="button"
-        className="Good__remove"
-      >
-        Remove
-      </button>
-
-      <button
-        type="button"
-        className="Good__select"
-      >
-        Select
-      </button>
-    </main>
-  );
+import goodsFromServer from './goods';
+
+type State = {
+  selectedGood: string,
 };
+
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    return (
+      <main className="App">
+        <header className="App__header">
+          <h1 className="App__title">
+            {this.state.selectedGood}
+            {' '}
+            is selected
+          </h1>
+          {
+            this.state.selectedGood !== 'No good'
+            && (
+              <button
+                type="button"
+                className="App__clear"
+                onClick={() => {
+                  this.setState({ selectedGood: 'No good' });
```

Comment:
'No good' is not a `selectedGood`

so if `selectedGood` will be `''` - render phrase 'No goods selected'
```suggestion
            this.state.selectedGood !== 'No good'
            && (
              <button
                type="button"
                className="App__clear"
                onClick={() => {
                  this.setState({ selectedGood: '' });
```